app/assets/javascripts/angular/controller/forms/updateForms.js.erb

```diff
@@ -165,7 +163,14 @@ app.controller('FormsUpdateCtrl', ['$scope', '$http', '$auth', '$location', '$ro
     });    
   }
 
-  //Drag and drop
+  //Set signature to true or false
+  $scope.signatureStatus = function() {
+    $http.patch(url + '/forms/' + $routeParams.id,$scope.form).success(function(response) {
```

Comment:
All I'm doing is updating the database to match the state of the checkbox, thats why there is nothing inside.